go/vt/vtctl/reparentutil/emergency_reparenter.go

```diff
@@ -717,26 +729,10 @@ func (erp *EmergencyReparenter) promoteNewPrimary(
 	tabletMap map[string]*topo.TabletInfo,
 	statusMap map[string]*replicationdatapb.StopReplicationStatus,
 ) error {
-	var err error
```

Comment:
This function now seems to be just 1 line. Is it called in multiple places? Can it be eliminated or combined with `reparentReplicas`? We may need a new name for the combined function, because it is effectively `promoteNewPrimaryAndReparentReplicas`.

Reply:
I guess it's actually `possiblyPromoteNewPrimaryAndDefinitelyReparentReplicas` 😁 I leave it to you to find a more elegant or concise name.

Reply:
I removed the function.